src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTask.java

```diff
@@ -68,9 +69,13 @@ public void run() {
 
                 repo.saveAndFlush(letter);
 
-                logger.debug("Marked letter {} as Uploaded", letter.getId());
-            } catch (Exception e) {
-                logger.error("Exception uploading letter {}", letter.getId(), e);
+                logger.debug("Marked letter {} as {}", letter.getId(), letter.getStatus());
+            } catch (FtpException exception) {
+                logger.error("Exception uploading letter {}", letter.getId());
+
+                throw exception;
+            } catch (IOException exception) {
+                logger.error(String.format("Failed to zip document for letter %s", letter.getId()), exception);
```

Comment:
we can't make this assumption. Either throw custom exception specific to zipping, or change the error message